quipucords/scanner/vcenter/tests_vc_inspect.py

```diff
@@ -227,22 +230,25 @@ def test_parse_vm_props(self, mock_dt):
 
             inspect_result = self.scan_task.inspection_result
             sys_results = inspect_result.systems.all()
-            expected_facts = {'vm.cluster': 'cluster1',
-                              'vm.cpu_count': 4,
-                              'vm.datacenter': 'dc1',
-                              'vm.dns_name': 'hostname',
-                              'vm.host.cpu_cores': 12,
-                              'vm.host.cpu_count': 2,
-                              'vm.host.cpu_threads': 24,
-                              'vm.host.name': 'host1',
-                              'vm.ip_addresses': ['1.2.3.4'],
-                              'vm.mac_addresses': ['00:50:56:9e:09:8c'],
-                              'vm.memory_size': 1,
-                              'vm.name': 'vm1',
-                              'vm.os': 'Red Hat 7',
-                              'vm.state': 'poweredOn',
-                              'vm.last_check_in': '2000-01-01 04:20:00',
-                              'vm.uuid': '1111'}
+            expected_facts = {
+                "vm.cluster": "cluster1",
+                "vm.cpu_count": 4,
+                "vm.datacenter": "dc1",
+                "vm.dns_name": "hostname",
+                "vm.host.cpu_cores": 12,
+                "vm.host.cpu_count": 2,
+                "vm.host.cpu_threads": 24,
+                "vm.host.name": "host1",
+                "vm.ip_addresses": ["1.2.3.4"],
+                "vm.is_template": True,
```

Comment:
I think its odd this "vm" is a template and also have ip/mac address.
I'd set the mock with "vm.is_template" = False just to avoid this.